inst/examples/RcppGibbs/timeRNGs.R

```diff
@@ -3,111 +3,36 @@ suppressMessages(library(Rcpp))
 suppressMessages(library(inline))
 suppressMessages(library(rbenchmark))
 
-## NOTE: This is the old way to compile Rcpp code inline.
-## The code here has left as a historical artifact and tribute to the old way.
-## Please use the code under the "new" inline compilation section.
-
-rcppGamma_old <- cxxfunction(signature(xs="numeric"), plugin="Rcpp", body='
-  NumericVector x(xs);
-  int n   = x.size();
-
-  // Initialize Random number generator
-  RNGScope scope;
-
-  const double y = 1.234;
-  for (int i=0; i<n; i++) {
-    x[i] = ::Rf_rgamma(3.0, 1.0/(y*y+4));
-  }
-
-  // Return to R
-  return x;
-')
-
-
-gslGamma_old <- cxxfunction(signature(xs="numeric"), plugin="RcppGSL",
-                        include='#include <gsl/gsl_rng.h>
-                                 #include <gsl/gsl_randist.h>',
-                        body='
-  NumericVector x(xs);
-  int n   = x.size();
-
-  gsl_rng *r = gsl_rng_alloc(gsl_rng_mt19937);
-  const double y = 1.234;
-  for (int i=0; i<n; i++) {
-    x[i] = gsl_ran_gamma(r,3.0,1.0/(y*y+4));
-  }
-  gsl_rng_free(r);
-
-  // Return to R
-  return x;
-')
-
-
-rcppNormal_old <- cxxfunction(signature(xs="numeric"), plugin="Rcpp", body='
-  NumericVector x(xs);
-  int n   = x.size();
-
-  // Initialize Random number generator
-  RNGScope scope;
-
-  const double y = 1.234;
-  for (int i=0; i<n; i++) {
-    x[i] = ::Rf_rnorm(1.0/(y+1),1.0/sqrt(2*y+2));
-  }
-
-  // Return to R
-  return x;
-')
-
-
-gslNormal_old <- cxxfunction(signature(xs="numeric"), plugin="RcppGSL",
-                        include='#include <gsl/gsl_rng.h>
-                                 #include <gsl/gsl_randist.h>',
-                        body='
-  NumericVector x(xs);
-  int n   = x.size();
-
-  gsl_rng *r = gsl_rng_alloc(gsl_rng_mt19937);
-  const double y = 1.234;
-  for (int i=0; i<n; i++) {
-    x[i] = 1.0/(y+1)+gsl_ran_gaussian(r,1.0/sqrt(2*y+2));
-  }
-  gsl_rng_free(r);
-
-  // Return to R
-  return x;
-')
-
-
 ## NOTE: Within this section, the new way to compile Rcpp code inline has been
-## written. Please use the code next as a template for your own project.
+## written. Please use the code next as a template for your own project, and
+## no NOT use the old code below
```

Comment:
*do NOT

Reply:
Fixed.  That was just a block I moved around so the typo has been there for years too :)